### PR TITLE
Add more tips for setting up environment

### PR DESCRIPTION
**Pull request by Jubilee101**
Some tips to help setting up the environment, as discussed [here](https://github.com/pgmoneta/pgmoneta/discussions/106)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Don't forget to indicate your pgmoneta version.
 You can use the follow command, if you are using a [Fedora](https://getfedora.org/) based platform:
 
 ```
-dnf install git gcc cmake make libev libev-devel openssl openssl-devel systemd systemd-devel zlib zlib-devel libzstd libzstd-devel lz4 lz4-devel libssh libssh-devel python3-docutils
+dnf install git gcc cmake make libev libev-devel openssl openssl-devel systemd systemd-devel zlib zlib-devel libzstd libzstd-devel lz4 lz4-devel libssh libssh-devel python3-docutils libatomic
```
Comment (jesperpedersen) [inline CONTRIBUTING.md:37]: Can you add `libcurl` and `libcurl-devel` as well

File: doc/tutorial/01_install.md [hunk 2]
```diff
@@ -143,11 +149,16 @@ Add the master key and create vault
...
 You have to choose a password for the master key - remember it !
 
+If you see an error saying `error while loading shared libraries: libpgmoneta.so.0: cannot open shared object` running 
+the above command. you may need to locate where your `libpgmoneta.so.0` is. It could be in `/usr/local/lib` or `/usr/local/lib64`
+depending on your environment. Add the corresponding directory into `/etc/ld.so.conf`, or alternatively, create a file
+called `pgmoneta_shared_library.conf` under `/etc/ld.so.conf.d/`, and add your directory into it. Remember to run `ldconfig` to make the change effective
+
 Create the `pgmoneta.conf` configuration
 
+Again, use `/usr/bin/` as `pgsql_dir` instead if you used dnf to install postgres
```
Comment (jesperpedersen) [inline doc/tutorial/01_install.md:161]: Did you mean `/usr/pgsql-10/bin/` here ?

**Review by jesperpedersen — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (Jubilee101) [reply CONTRIBUTING.md]: Added

File: doc/tutorial/01_install.md (hunk 2 above)
Comment (Jubilee101) [reply doc/tutorial/01_install.md]: Yes, I elaborate on this in the newest commit, unless you mean I'm not supposed to replace it?

File: CMakeLists.txt [hunk 3]
```diff
@@ -1,134 +1,87 @@
...
-  else ()
-    message(FATAL_ERROR "systemd needed")
-  endif()
-endif()
-
-file(MAKE_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/src/")
-
-add_subdirectory(doc)
-add_subdirectory(src)
+cmake_minimum_required(VERSION 3.24)
```
Comment (jesperpedersen) [inline CMakeLists.txt:1]: This is an unrelated change

**Review by jesperpedersen — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 3 above)
Comment (jesperpedersen) [reply CMakeLists.txt:1]: Remove the `CMakeLists.txt` change, then squash and force push

Comment (Jubilee101) [reply CMakeLists.txt]: Done. So sorry about that, I'm using CLion locally and it won't let me use ctrl + click to jump to definitions unless it configures the cmakelist, which is based on linux environment. So I had to make my own cmakelist and forgot to restore it when pushing.

Comment (jesperpedersen) [reply CMakeLists.txt]: https://www.scraggo.com/how-to-squash-commits/

Comment (Jubilee101) [reply CMakeLists.txt]: Done, I thought I just need to squash the last 2 commits :), the cmakelist fix and the elaboration one.

**Comment by jesperpedersen** [comment]
Ah, I forgot -- add yourself to `AUTHORS` -- commit, squash and force push again.

Otherwise it looks good !

**Comment by Jubilee101** [comment]
Add myself to authors :)

**Comment by jesperpedersen** [comment]
You need to rebase in order to resolve the conflict

**Comment by jesperpedersen** [comment]
Could you rebase and resolve the conflict ?

**Comment by Jubilee101** [comment]
So sorry I missed the message, I think the conflict is resolved now.

**Comment by jesperpedersen** [comment]
Rebased, and merged.

Thanks for your contribution !
